lib/emque/producing/message/message_with_changeset.rb

```diff
@@ -0,0 +1,60 @@
+module Emque
+  module Producing
+    module MessageWithChangeset
+      include Emque::Producing::Message
+
+      def self.included(base)
```

Comment:
@semmons99 - Question about this when you get a chance. I ended up having to revert back to doing the `base.extend` here as it seemed like the `ClassMethods` from `Emque::Producing::Message` were making it to `MessageWithChangeset`, but not then getting further pushed down into, for instance, `TestMessageWithChangeset` from the specs.

For reference, we were originally assuming we wouldn't need to have any of this since we were including `Emque::Producing::Message` which we expected to have handled this.